components/Starknet/modules/starkgate/pages/overview.adoc

```diff
@@ -14,15 +14,15 @@ include::partial$snippet_backwards_compatibility_note.adoc[]
 [#starkgate_addresses]
 == StarkGate addresses
 
-L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files in the Starknet GitHub repository shown in the table xref:#table_StarkGate_token_addresses[].
+L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files shown in the table xref:#table_StarkGate_token_addresses[].
```

Comment:
Please also remove the "in the Starknet GitHub repository" part in the second paragraph of "Supported tokens in StarkGate" (currently line 33) as well.